kuma/payments/jinja2/payments/thank_you.html

```diff
@@ -100,7 +100,8 @@ <h3>{{_('Refunds and cancelling your payments')}}</h3>
                                 {% trans url="mailto:" + settings.CONTRIBUTION_SUPPORT_EMAIL + "?Subject=Cancel%20monthly%20payment",
                                     email=settings.CONTRIBUTION_SUPPORT_EMAIL %}
                                     If you would like to cancel your monthly payment or apply for a refund, you are free to do so at any point.
-                                    Please contact <a href="{{ url }}">{{ email }}</a> to cancel and we won’t charge you for the following month.
+                                    Please contact <a href="{{ url }}">{{ email }}</a> to cancel. If you choose to cancel, we will not charge your payment
```

Comment:
Is "subsequent months" more legally precise than "the following month"? Feels like a $10 word when a $0.10 word would work. I know, adding to the copy document.

Reply:
After discussing with our lawyer, "subsequent months" seems better than the longer "starting next month and the following months". Precise English wins.